.github/workflows/rust.yml

```
@@ -78,13 +76,11 @@
         uses: actions/checkout@v4
       - name: Setup Rust toolchain
         uses: dtolnay/rust-toolchain@stable
-        with:
-          toolchain: stable
           components: clippy
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 79: mapping values are not allowed in this context

(syntax-check)